src/System.IO.Compression/src/System/IO/Compression/DeflateStream.cs

```diff
@@ -209,6 +209,14 @@ public override void Flush()
             return;
         }
 
+        public override Task FlushAsync(CancellationToken cancellationToken)
+        {
+            EnsureNotDisposed();
```

Comment:
Thanks Stephen sorry I missed that explanation in the commit message.  Yes I agree, talking to Eric also since this is a good behavior change and is unlikely to be hit by correct code, we can quirk later if it's a problem.